src/Microsoft.ML.Data/Evaluators/EvaluatorUtils.cs

```diff
@@ -934,7 +934,7 @@ public static IDataView ConcatenateOverallMetrics(IHostEnvironment env, IDataVie
                                  variableSizeVectorColumnName, vectorType);
 
                         // Drop the old column that does not have variable length.
-                        idv = ColumnSelectingTransformer.CreateDrop(env, idv, variableSizeVectorColumnName);
+                        idv = idv = ColumnSelectingTransformer.CreateDrop(env, idv, variableSizeVectorColumnName);
```

Comment:
>idv = idv =  [](start = 24, length = 12)

fix

Reply:
Thank you for catching this!

---
In reply to: [254361008](https://github.com/dotnet/machinelearning/pull/2367#discussion_r254361008) [](ancestors = 254361008)